### PR TITLE
[mp] new docs overview

### PR DESCRIPTION
**Pull request by mattppal**
# Description

Preview: https://mage-staging.mintlify.app/introduction/overview

- Simplify overview page
- Record new product tour gifs
- Update messaging
- Update HTML for shields

I feel pretty strongly that simpler is better on an overview page. That was my general approach here. I highly recommend running this locally to see the new format

# How Has This Been Tested?

- [x] `mintlify dev`
- [x] `mintlify broken-links`

image

# Checklist
- [x] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: docs/introduction/overview.mdx [hunk 1]
```diff
@@ -153,356 +20,114 @@ description: "🧙 A modern replacement for Airflow."
...
-or
+Have you met anyone who loves developing in Airflow? That's why we designed an easy developer experience that you'll enjoy.
 
-```bash
-conda install -c conda-forge mage-ai
-```
+- **Simple developer experience**: Start developing locally with a single command or launch a dev environment in your cloud using Terraform.
+- **Language of choice**: Write code in Python, SQL, or R in the same pipeline for _ultimate_ flexibility.
+- **Engineering best practices built-in**: Every step in your pipeline is a standalone file with modular code that's reusable and testable. No more DAGs with spaghetti code 🍝.
+- **Bring your favorite tools:** Write [dbt](dbt/overview) models, use your favorite [IDE](development/text-editor), [track changes with Git](getting-started/setting-up-git), and much much more.
```
Comment (wangxiaoyou1993) [inline docs/introduction/overview.mdx:45]: Could you use the absolute paths in the url instead of the relative paths?
The links don't work on staing.

Comment (wangxiaoyou1993) [reply docs/introduction/overview.mdx:45]: Some of the links are not broken links. They just redirect to the overview page. Are we able to catch them with `mintlify broken-links`?

File: docs/introduction/overview.mdx [hunk 2]
```diff
@@ -153,356 +20,114 @@ description: "🧙 A modern replacement for Airflow."
...
-
-Check out our
-[FAQ page](https://www.notion.so/mageai/Mage-FAQs-33d93ee65f934ed39568f8a4bc823b39)
-to find answers to some of our most asked questions.
-
----
+> A group of mages who help each other realize their full potential! Join us on [Slack](https://www.mage.ai/chat).
 
-## 🪪 License
+<p align="center">
```
Comment (wangxiaoyou1993) [inline docs/introduction/overview.mdx:120]: Are we able to hide the other channels in this image?

File: docs/introduction/overview.mdx [hunk 3]
```diff
@@ -153,356 +20,114 @@ description: "🧙 A modern replacement for Airflow."
...
-
-<Card
-  title="GitHub"
-  icon="github"
-  href="https://github.com/mage-ai/mage-ai"
-></Card>
-
----
-
-## 🤔 Frequently Asked Questions (FAQs)
```
Comment (wangxiaoyou1993) [inline docs/introduction/overview.mdx:490]: Are we deprecating the FAQ page?

Comment (mattppal) [reply docs/introduction/overview.mdx:490]: It's a on notion. I think we should move it to mintlify. I should be able to knock that out in about 30m or so, I can make it a follow-up

File: docs/introduction/overview.mdx (hunk 2 above)
Comment (mattppal) [reply docs/introduction/overview.mdx:120]: Good call, I'll recreate :)

File: docs/introduction/overview.mdx (hunk 1 above)
Comment (mattppal) [reply docs/introduction/overview.mdx:45]: Fixed, this was a backslash issue

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**